docs/spec/modals/modals.directive-examples.nested-modals.use-cases.md

```diff
@@ -0,0 +1,34 @@
+7.14: Modals directive examples nested modals
+===============================================
+
+**Primary Actor**: User  
+
+**Scope**: Ngx-bootstrap DEMO / BS version 4/ v.3.0.1
+
+**Goal**: Show user Child modal functionality
+
+Main success scenario:
+----------------------
+
+1. User opens 'Modals' demo page
+2. User clicks on 'Nested modals' sub-menu
+3. After click on "Open parent modal" button,  modal popup is opened and user sees 'Open second modal' button
```

Comment:
"Open parent modal" button - need to use same format for quotes, I'm using " at all places and recommend you too)